fabric_cf/actor/test/core/policy/broker_simpler_units_policy_test.py

```diff
@@ -121,8 +122,12 @@ def build_sliver(self) -> NodeSliver:
         node_sliver.node_id = "test-slice-node-1"
         cap = Capacities()
         cap.set_fields(core=4, ram=64, disk=500)
+        catalog = InstanceCatalog()
+        instance_type = catalog.map_capacities_to_instance(cap=cap)
+        cap_hints = CapacityHints().set_fields(instance_type=instance_type)
         node_sliver.set_properties(name="node-1", type=NodeType.VM, site="RENC",
-                                   capacities=cap, image_type='qcow2', image_ref='default_centos_8')
+                                   capacities=cap, image_type='qcow2', image_ref='default_centos_8',
```

Comment:
This test sets both capacities and capacity_hints - which is preferred in your code today (in request setting both is redundant and one should be ignored).

Reply:
This should have been a question: which is preferred in  your code? Do you first look at capacities and if set, ignore hints, or look at hints and if set, ignore capacities?

Reply:
This is related to comment below